staging/src/k8s.io/apiextensions-apiserver/pkg/registry/customresource/strategy.go

```diff
@@ -271,3 +279,13 @@ func (a customResourceStrategy) MatchCustomResourceDefinitionStorage(label label
 		GetAttrs: a.GetAttrs,
 	}
 }
+
+// OpenAPIv3 type/maxLength/maxItems/MaxProperties/required/wrong type field validation failures are viewed as blocking err for CEL validation
+func hasBlockingErr(errs field.ErrorList) (bool, *field.Error) {
+	for _, err := range errs {
+		if err.Type == field.ErrorTypeRequired || err.Type == field.ErrorTypeTooLong || err.Type == field.ErrorTypeTooMany || err.Type == field.ErrorTypeTypeInvalid {
+			return true, field.Invalid(nil, nil, "some validation rules were not checked because the object was invalid; correct the existing errors to complete validation")
```

Comment:
Hello, using nil field name and value as a placeholder produces confusing message, e.g.:
```
The FooType "foo" is invalid:
* spec.bar: Too long: may not be longer than 63
* <nil>: Invalid value: "null": some validation rules were not checked because the object was invalid; correct the existing errors to complete validation
```


Would it be OK if we change this to reuse field path and type from the `err` that error message looks like:
```
The FooType "foo" is invalid:
* spec.bar: Too long: may not be longer than 63
* spec.bar: Too long: some validation rules were not checked because the object was invalid; correct the existing errors to complete validation
```

Ideally this should be not a filed error but object error but that is not supported afaict.